api/tools/src/main/java/org/pmiops/workbench/tools/cdrConfig/test/README.md

```diff
@@ -0,0 +1,29 @@
+# Test files for UpdateCdrConfig
```

Comment:
Should this be in the `test` directory instead? Possibly under `resources`? e.g. in here https://github.com/all-of-us/workbench/tree/c82446df285cca307856c8a688489501793b4289/api/src/test/resources (or maybe in a parallel "tools" test dir..?)